app/bundles/LeadBundle/Views/List/list.html.php

```diff
@@ -171,7 +171,10 @@
                             <?php if ($lastBuiltDateDifference >= $segmentRebuildWarningThreshold): ?>
                                 <label class="control-label" data-toggle="tooltip"
                                        data-container="body" data-placement="top" title=""
-                                       data-original-title="<?php echo $view['translator']->transChoice('mautic.lead.list.form.config.segment_rebuild_time.message', $lastBuiltDateDifference, ['%hours%' => $lastBuiltDateDifference]); ?>">
+                                       data-original-title="<?php echo $view['translator']->trans(
+                                               'mautic.lead.list.form.config.segment_rebuild_time.message',
+                                               ['%hours%' => $lastBuiltDateDifference]
```

Comment:
```suggestion
                                               ['%count%' => $lastBuiltDateDifference]
```
plus update the ini file